hardware/Drivetrain.java

```diff
@@ -231,7 +233,7 @@ public void turn(int degrees, double power) {
         resetAngle();
     }
 
-    public void setSlow() {
-        slow_mode = !slow_mode;
-    }
+    public void setSlow() { slow_mode = true; }
+
+    public void unSlow() { slow_mode = false; }
```

Comment:
better method names. these are probably still kind of redundant in my opinion? can't they just be combined into one with a not operator? maybe it was because the not was just mapped to one button. idk test this more